### ISSUE TITLE
Calls to FCL

### ISSUE THREAD

**Issue by jmirabel** [humanoid-path-planner/hpp-pinocchio#19]
With the API of hpp-model, a distance request to FCL was:
```cpp
fcl::distance(obj1->fcl(), obj2->fcl(), request, result)
```
With hpp-pinocchio, we have:
```cpp
fcl::distance(obj1->fcl().collisionGeometry().get(), obj1->getFclTransform(), 
              obj2->fcl().collisionGeometry().get(), obj1->getFclTransform(),
              request, result)
```
which is not very convenient.

This is because pinocchio do not use the transformation inside the `fcl::CollisionObject` (I think but I'm not sure). The API of `CollisionObject` to get a more handy and readable code.

**Comment by nmansard** [comment]
Related to stack-of-tasks/pinocchio#253

**Comment by nmansard** [comment]
Solved by stack-of-tasks/pinocchio#280
